backend/env-vars.ftd

```diff
@@ -33,5 +33,116 @@ lang: sh
 
 FASTN_DANGER_ACCEPT_CHECKED_IN_ENV=true fastn serve
 
+-- ds.h1: Supported Environment Variables
+
+`fastn` supports the following environment variables:
+
+-- ds.h2: Postrgres variables
+
+-- fastn-pg-url:
+-- fastn-pg-danger-disable-ssl:
+-- fastn-pg-ssl-mode:
+-- fastn-pg-danger-allow-unverified-certificate:
+-- fastn-pg-certificate:
```

Comment:
Let's wrap them in another component called pg-variables or something, so you do not have to duplicate this list, which also ensures we never forget to add something in pg.ftd when we update env-vars.ftd.